docs: merge docs from 1 to 2 (#16505)
appium/appium: docs/README.md

## doc translation

> At the moment we are focusing on improving the english documentation. When it
> is done we are planning to setup proper i18n l10n processes so that it is

Reviewer: ```suggestion
At the moment we are focusing on improving the English documentation. When it
is done, we are planning to establish proper i18n and l10n processes so that it is
```

easier for people to help in translating the documentation.

## Bump supported NodeJS version

> - Refer to https://nodejs.org/en/download/releases/ if current supported version is EOL

Reviewer: ```suggestion
- Refer to the [Node.js Release Working Group README](https://github.com/nodejs/release#readme) to check LTS, maintenance, and EOL Node.js versions
```

> - Example to bump supported NodeJS version: https://github.com/appium/appium/pull/15000

Reviewer: ```suggestion
- Example to bump supported NodeJS version: appium/appium#15000
```
